### DISCUSSION TITLE
Améliorer la page d'accueil

### DISCUSSION THREAD

**Discussion by hfroot** [betagouv/ma-cantine#1002] [Idées]
En fonctionne de personas créés.

**Comment by alemangui** [comment]
@JenniferStephan a commencé une réflexion dessus [sur Figma](https://www.figma.com/file/Wy0JohuTd3zP9fTGERs5qq/Egalim-nouveauparcours?node-id=1%3A3)

**Reply by JenniferStephan** [reply]
Merci d'avoir créé cette discussion. 
J'ai l'impression, qu'avant de parler de refonte totale de la landing; on peut déjà continuer le travail que tu avais fait @hfroot en changeant, dans la bannière qui est censée décrire notre mission, changer notre tagline et CTA, avec comme tu dis, en tête, les besoins et bénéfices que l'on apporte aux users. 

Mes propositions : 
**Tagline :** "Mieux manger de la crêche à l’EHPAD" : plus large que l'actuelle tagline qui se focus sur la loi, parle de diagnostic de manière jargonneuse et exclus les convives qui sont un de nos personae importants

**sous-titre :** "ma-cantine accompagne la transition alimentaire de la restauration collective en outillant les gestionnaires afin de fomenter les bonnes pratiques tant en matière de santé que de respect de l’environnement et en informant les convives sur le contenu de leur assiette !"

Là je pense que l'on peut faire mieux niveau rédaction... J'aimerais introduire les deux notions : 
- transparence des assiettes
- si et seulement si on outille les gestionnaires

en lien avec proposition de **CTAs**

1 > Découvrir le réseau de cantines 
qui renvoie vers /nos-cantines

2 > Me créer un compte/m'identifier gestionnaire
qui renvoie vers création de compte

<img width="857" alt="Capture d’écran 2022-02-07 à 13 30 13" src="https://user-images.githubusercontent.com/36134318/152789200-76c6d2cd-8c5c-425e-acf9-ed18ccbd7336.png">

j'embarquerais bien ces premiers changements sur le prochain sprint; mais à discuter niveau évolution /modifs des textes

**Reply by valeriemerle** [reply]
Plutot ok sur cette nouvelle présentation plus dynamique, mieux ciblée, plus claire de la plateforme.  Par contre sur le "wording" je remplacerai le terme de "fomenter" par "susciter" ou "encourager" plus positif dans son acception. Plus à discuter : les images moins de protéines animales et de produits transformés peut etre...? proposer aussi de mettre en valeur la page "chiffres clés" mais en la séparant de "nos cantines" pour eviter la confusion entre cantines volontaires qui ont publié et la base de données totale de ma cantine qui permet d'agréger les info.

**Reply by hfroot** [reply]
Merci Jenn pour la belle maquette !
Je suis hésitante à trop axer le design autour des personas qu'on a créer. Bien sûr chez nous ça aide à décider quoi construire, mais:
1. une personne peuvent être plusieurs choses (une parent et une gestionnaire/élue etc.)
2. si on crée nouveaux personas, cette page pourrait devenir trop bruyante

Je me permet à suggérer un format alternatif :

![Screenshot 2022-02-09 at 13 06 23](https://user-images.githubusercontent.com/9282816/153198789-7df0025f-87a4-44e6-863d-30c26f956a07.png)

où le lien 'en savoir plus' envoie l'usager plus bas sur la page :

![Screenshot 2022-02-09 at 13 06 35](https://user-images.githubusercontent.com/9282816/153198799-fee7cb26-7477-4904-a8a4-f12d84186be9.png)

Cette deuxième partie vise répondre à trois retours de "manque des infos avant commencer le démarche" qui j'ai trouvé dans les retours "je donne mon avis".

Tu penses quoi ?

**Reply by alemangui** [reply]
Hmm... de mon coté j'aime bien le layout side-by-side de la proposition de Jenn car ça nous permet de montrer tous les cas d'usage sans scroller. De toute façon dans les petits écrans on devra mettre les uns au dessus des autres. @hfroot si jamais on a des personas additionnels on devra réfléchir à un layout alternatif en effet, mais pas certain que cela arrive.

Par contre partant pour discuter avec Anne-So sur la façon de présenter ces trois cas : soit en spécifiant la persona (comme @JenniferStephan suggère), soit en spécifiant l'action à prendre ou la question à résoudre (comme @hfroot suggère). Intuitivement je trouve plus clair la proposition d'Helen.

Je vous propose :
1- On commence à construire le layout et le wording de la story
2- On voit avec Anne-So des éventuels changements de wording suite à la suggestion de @hfroot (thread spécifique ici : https://github.com/betagouv/ma-cantine/discussions/1002#discussioncomment-2148820)
3- La section _En savoir plus_ proposée par @hfroot me semble répondre à une problématique spécifique et mérite peut-être sa propre réflexion et story ? 

Vous en pensez quoi ?

**Comment by alemangui** [comment]
Je me permet de créer un thread à part (en complément à [mon commentaire](https://github.com/betagouv/ma-cantine/discussions/1002#discussioncomment-2148607)) par rapport à la [suggestion](https://github.com/betagouv/ma-cantine/discussions/1002#discussioncomment-2140892) de @hfroot de changer le wording.

Ça me semble en effet plus intuitif d'axer les textes par rapport à ce que l'outil résout comme problème. Par exemple, au lieu d'avoir un bouton _"Accéder au tableau de bord élu"_ on peut dire ce que la personne trouvera, par ex _"Voir les chiffres clés par territoire"_.

Est-ce que ça aurait du sens également, comme suggère Helen, de changer les _"je suis..."_  par _"je veux..."_ ? Genre _"Je suis convive et parent de convive"_ par _"Je veux savoir où en est l'établissement ou je mange"_ (pas top mais bon c'est pour donner une idée). Ça rejoint une remarque que @astranchet avait fait par rapport aux termes jargonneux dont "convive".

On voit Anne-So jeudi prochain, mais déjà on peut commencer à réfléchir de notre coté pour mieux profiter de notre réu avec elle.

**Comment by alemangui** [comment]
CR de la réunion avec Anne-So concernant la landing page : 

### Wording
- Éviter les "je suis" ou "je veux" car difficile de trouver la partie différentiante
- Éviter le jargon pour tout ce qui est à destination des convives. En revanche, le jargon technique peut être utilisé pour les pros (gestionnaires et élus) tant qu'iels soient habitués à ces termes et que les comprennent bien.
- Nos phrases sont trop longues. La landing page n'a pas vocation à être exhaustive car son rôle est d'accrocher et rediriger. Nos phrases doivent être courtes et avec un verbe d'action. L'outil [Hemingway](https://hemingwayapp.com/) indique quand une phrase est trop longue et donc à racourcir (couleurs jaune et rouge). Chaque mot compte.
- Dans la section pour les trois personnas : parler de l'objectif final, par ex privilégier "Étudier l'évolution de votre territoire" plutôt que "Je suis élu/élue"
- Dans la section pour les trois personnas : la phrase et le bouton son redondantes - s'inspirer de [le.taxi](https://le.taxi/) dans lequel on a une carte cliquable sans bouton.
- Prendre en compte qu'on ne lit pas dans l'ordre dans le web. Dans notre cas les gens liront : titre, bouton, texte au milieu.
- Le texte de sous-titre est trop long et comprend des mots un peu compliqués à interpreter (par ex. "fomenter"). Peut-être reformuler en bullet-points très concis comme [le.taxi](https://le.taxi/)
- Les deux boutons principales sont redondants avec ceux dans la section pour les trois personnas. Ça avait du sens à l'époque où on avait une seule personna/proposition de valeur, mais maintenant qu'on en a plus, on peut les enlever (voir [le.taxi](https://hemingwayapp.com/), dans lequel la première partie de leur landing n'a pas des éléments cliquables).

### UI
- On peut utiliser une bordure ou une couleur de fond pour séparer les sections et faciliter la visibilité. Possibilité aussi de faire des cartes pour les trois éléments de la section personnas
- Le sous-titre de la section principale doit avoir une taille de police plus importante que les sous-titres de la section personnas.
- Avoir une vidéo aesthétique de fond n'est pas très intéressante car a) ça n'apporte pas d'info, b) pas top niveau a11y (contrastes changeants, mouvement, distraction), et c) trop de ressources pour pas grand gain (interdit d'ailleurs dans le DSFR par des considérations d'éco-conception et a11y)
- Laisser seulement les actions des trois cartes personnas.
